src/Tribe/Attendees.php

```diff
@@ -538,6 +538,11 @@ private function generate_filtered_list( $event_id ) {
 				// should simply be passed back unmodified
 				$row[ $column_id ] = $this->attendees_table->column_default( $single_item, $column_id );
 
+				// In the case of orphaned field data, handle converting array to string.
+				if ( is_array( $row[ $column_id ] ) && array_key_exists( 'value', $row[ $column_id ] ) ) {
```

Comment:
Why array key exists and not `isset`?
We might be able to just use `Arr::get()` here.

Reply:
I found that when using `isset()` it returns false when the value is null, even though the key exists. `array_key_exists()` works, regardless. I'll test with `Arr::get()` to see if that works better.

Reply:
Good call on using `Arr::get()`. It works perfectly.